tests/unit_tests/test_suite_test.go

```diff
@@ -0,0 +1,89 @@
+package tests
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"kubevirt.io/client-go/kubecli"
+
+	instancetypev1beta1 "kubevirt.io/api/instancetype/v1beta1"
+)
+
+const (
+	testNamespace = "common-instancetype-functest"
+
+	clusterInstanceTypesBundlePath = "../../_build/common-clusterinstancetypes-bundle.yaml"
+	clusterPreferencesBundlePath   = "../../_build/common-clusterpreferences-bundle.yaml"
+)
+
+var (
+	virtClient kubecli.KubevirtClient
+
+	virtualMachineClusterInstancetypes []instancetypev1beta1.VirtualMachineClusterInstancetype
+	virtualMachineClusterPreferences   []instancetypev1beta1.VirtualMachineClusterPreference
```

Comment:
Maybe these need to be reset after every test? They are mutable.

Reply:
I am only reading them, so I think it should be fine. Although it could be easily reset by moving the `loadBundles` function into `BeforeEach`, so let me know if you still want to reset or not.

Reply:
Please reset them, since they are mutable a test could change them. To avoid loading every test iteration you could also keep a copy of the initial slice.

Reply:
Ok